go.mod

```diff
@@ -0,0 +1,73 @@
+module sigs.k8s.io/sig-storage-local-static-provisioner
+
+go 1.14
```

Comment:
Should this be 1.13?

Reply:
reverted

I originally wanted to upgrade go to 1.14, but the prow image uses 1.13.